### PR TITLE
Fix/logster install

### PR DESCRIPTION
**Pull request by Mossman1215**

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -1,110 +1,131 @@
...
+  if $facts['lsbdistcodename'] == 'buster' {
+    file { 'git_proxy':
+      path => '/root/.gitconfig',
+      content => "[http] \n proxy = ${::params::http_proxy}",
+      before => Exec['installLogster']
+    }
+    $logster_vcs = 'https://github.com/etsy/logster.git'
+    $logster_ref = 'a24249886391a8b885d21c882f6fcaa95e29b015'
+    exec { 'installLogster':
+      command => "pip install --proxy ${::params::http_proxy} git+${logster_vcs}@${logster_ref}#egg=logster",
```
Comment (McJoppy) [inline manifests/install.pp:25]: Take a look at stdlib

https://ask.puppet.com/question/25819/how-to-install-pip-packages/

Should be able to manage proxy better too

Comment (McJoppy) [reply manifests/install.pp:25]: Just tested - the logster package from stretch and jessie installs fine on buster too.

Minimal dependencies:
`Depends: logtail, python-pkg-resources, python:any (>= 2.7.5-5~), python:any (<< 2.8), python`

Comment (Mossman1215) [reply manifests/install.pp:25]: i think pulling in the logster deb package makes sense to me

Comment (Mossman1215) [reply manifests/install.pp:25]: cloning requires github to be up so that's not ideal to have puppet runs fail just because github is down

**Comment by Mossman1215** [comment]
Seems to be working on buster now (tested on awstest1)
`Notice: /Stage[main]/Mariadb/Exec[mariadb-sysmaint-fix]/returns: executed successfully (corrective)
Notice: /Stage[main]/Ss_graphite_client::Install/File[/opt/logster/logster_0.0.1-2_all.deb]/ensure: defined content as '{md5}dd3a6ae6383bfbc4c167e10ad54f4107'
Notice: /Stage[main]/Ss_graphite_client::Install/Package[logster]/ensure: created
`

File: manifests/install.pp [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+  if $facts['lsbdistcodename'] == 'jessie' {
+    package {
+    'realpath': ensure => installed;
     'logster': ensure => installed;
+    }
+  }
+  #logster no longer included as of buster
+  #so we can install via the stretch deb (last possible debian package version)
+  if $facts['lsbdistcodename'] == 'buster' {
+    file { '/opt/logster/logster_0.0.1-2_all.deb':
```
Comment (McJoppy) [inline manifests/install.pp:17]: `/opt/logster` doesn't exist - and opt is generally used to install optional software.

Can you please transfer and install the logster `.deb` file to and from `/usr/src` as a more suitable location.
eg. `file { '/usr/src/logster_0.0.1-2_all.deb':`

Comment (Mossman1215) [reply manifests/install.pp]: Yup that's now amended to be `/usr/src`

**Pull request merged**